db/seeds.rb

```diff
@@ -10,18 +10,9 @@
                password_confirmation: password)
 end
 
-# generate data for statuses
-Status.create(title: "Done", background: "#64d974")
-Status.create(title: "In progress", background: "#07c0e3")
-Status.create(title: "Ongoing", background: "#6d28f1")
-Status.create(title: "Unknown", background: "#b3a8a9")
-
-
 # Generate microposts for a subset of users.
```

Comment:
Eredetileg csak ehhez a kódrészlethez szerettem volna kommentet írni, de az egész seed file nagyon csúnya. Légyszi legyen ennek a PR-nak a része, hogy rendbeszedjük ezt a file-t, úgyis kb a feléhez hozzányúltunk már.

- A fájlban megtalálhatók az ősi ellenségeim a kommentek. Itt nem rossz, hogy vannak kommentjeink, viszont egy praktikusabb megoldás, ha meg is osztjuk a fejlesztővel ezeket a fájl futása közben.

```
# generate data for statuses
```

komment helyett mindenhol legyen egy konzolra írás

```ruby
puts 'creating statuses'
```

- A kommentek nem tükrözik a valóságot. "Generate microposts for a subset of users." Tudtommal nincsenek micropost-jaink. Ezeket aktualizáljuk légyszi!

- (Írnék kommentet az egyszer használatos változókra, de a következő komment ezt feloldja.)

- Nem érdemes újra feltalálni a kereket. Jelenleg rendelkezünk factory-kkal, amik tisztában vannak vele, hogy hogyan kell mintaadatokat létrehozni. Illetve rendelkezünk segédfüggvénnyel, aminek a segítségével fix számú mintarekordot tudunk létrehozni.

```ruby
5.times do |n|
  first_name  = Faker::Name.first_name
  last_name  = Faker::Name.last_name
  email = "example-#{n+1}@beloved.org"
  password = "password"
  User.create!(first_name:  first_name,
               last_name: last_name,
               email: email,
               password:              password,
               password_confirmation: password)
end
```

egy ilyen helyett valami hasonlót szeretnék látni:

```ruby
FactoryBot.create_list :user, 3
```

- Illetve egy tök hasznos dolog, ha van egy fix felhasználónk a seed file-ban, aki mindig létrejön. Így el lehet menteni localhost-ra bejelentkezéshez.

```ruby
puts 'creating VIP' # VIP helyett, ha van pl admin és kliens jogosultságunk, akkor érdemes egy-egy ilyen user-t létrehozni "creating admin" & "creating client" etc. You get the idea.
FactoryBot.create :user, email: 'that@specific.email', password: 'somethingSENSIBL3' # megadhatod, hogy mindig ugyanaz legyen a neve is. alapvetően a credentials fontos, hogy mindig ugyanaz legyen
```

Reply:
Utánanéztem, mert szerettem volna neked rosszaló kommentet írni arra, hogy a `project` factory-ja nem változott a PR-ban. Sadly, nem rendelkezünk project factory-val. :upside_down_face: 

Az lenne a feladat, hogy hozzuk létre ezt a Factory-t.
- Ihletet tudsz meríteni a `spec/factories/users.rb` -ből.
- Mi a `spec/factories/projects.rb` -t szeretnénk létrehozni.
- Légyszi a fájl első sorában legyen `# frozen_string_literal: true`
- Bármilyen `Faker` helper-t használhatsz a `project.title` generálásához:
  - `Faker::Lorem.word`
  - `Faker::Movie.title`
  - `Faker::Hacker.say_something_smart`
  - A `Faker` doksit megtalálod többek között itt is: https://www.rubydoc.info/github/faker-ruby/faker/Faker (a baloldali menüben tudsz majd navigálni)